datumaro/plugins/mpii_format.py

```diff
@@ -0,0 +1,128 @@
+# Copyright (C) 2021 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import json
+import os.path as osp
+
+import numpy as np
+
+from datumaro.components.annotation import Bbox, Points, PointsCategories
+from datumaro.components.extractor import (
+    AnnotationType, DatasetItem, Importer, SourceExtractor,
+)
+from datumaro.components.format_detection import FormatDetectionContext
+from datumaro.components.media import Image
+
+
+class MpiiPath:
+    ANNOTATION_FILE = 'mpii_annotations.json'
+    HEADBOXES_FILE = 'mpii_headboxes.npy'
+    VISIBILITY_FILE = 'jnt_visible.npy'
+    POS_GT_FILE = 'mpii_pos_gt.npy'
+
+MpiiPointsCategories = [
```

Comment:
Please use UPPER_CASE_WITH_UNDESCORES for constants. It's not a class or enum.

Reply:
Fixed.